dev-vcs/git-remote-hg/git-remote-hg-1.0.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=(python2_7)
+
+DESCRIPTION="Semi-official Mercurial bridge from Git project"
+HOMEPAGE="https://github.com/mnauw/git-remote-hg"
+SRC_URI="https://github.com/mnauw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+CDEPEND="dev-vcs/git
+	dev-vcs/mercurial"
+
+DEPEND="${CDEPEND}
+	app-text/asciidoc"
+RDEPEND="${CDEPEND}"
+
+# A few tests fail.
+RESTRICT="test"
+
+src_install() {
+	dobin git-remote-hg
```

Comment:
Why are you installing it manually rather than using setup.py? Also, it indicates you're not installing the complete set.

Reply:
I'm not using setup.py (through distutils eclass) because:

    python2.7 setup.py build
    fatal: not a git repository (or any parent up to mount point /var/tmp)
    Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

Also, the upstream documentation states that installation is performed with:

    wget https://raw.github.com/mnauw/git-remote-hg/master/git-remote-hg -O ~/bin/git-remote-hg
    chmod +x ~/bin/git-remote-hg

which in an ebuild translates into a simple `dobin`.

Reply:
OK, the Makefile actually works. I switched to `emake` for installing. This also creates and installs the man page.